src/map/skill.cpp

```diff
@@ -1394,6 +1399,13 @@ int skill_additional_effect(struct block_list* src, struct block_list *bl, uint1
 #endif
 		break;
 
+	case NPC_STORMGUST2:
+		if (skill_lv < 3)
+			sc_start(src,bl,SC_FREEZE,10-((skill_lv-1)*(skill_lv+1)),skill_lv,skill_get_time2(skill_id,skill_lv));	// 10;7;3 % chance
```

Comment:
Could we make that more readable?
I kinda dont understand why a formula like this is necessary for 2 values... :(

Reply:
Alright (it's a clumsy formula for the 10;7;3 % chance per skill level)

Reply:
Just make it a short if with the 3 values or if/else if/else then it will be more readable I guess.